datalad/support/tests/test_annexrepo.py

```diff
@@ -615,52 +615,52 @@ def test_AnnexRepo_commit(src, path):
         ok_clean_git(path, annex=True)
 
 
-@assert_cwd_unchanged
-@with_testrepos('.*annex.*', flavors=local_testrepo_flavors)
-@with_tempfile
-def test_AnnexRepo_add_to_git(src, dst):
-
-    ds = AnnexRepo(dst, src)
-
-    filename = get_most_obscure_supported_name()
-    filename_abs = opj(dst, filename)
-    with open(filename_abs, 'w') as f:
-        f.write("What to write?")
```

Comment:
ideally we should have test like this.... especially since it is testing using obscure filename here... sorry if I have missed if such testing moved somewhere else